Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
@clytwynec That's a good point. I don't know whether we can generalize the cookiecutter repo so that it doesn't assume Python/Django, or whether it makes sense to introduce another repo. I don't particularly like having hundreds of small repos, so my preference would be to keep it within one of the existing ones.